config/deploy.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-env node */
```

Comment:
Why this addon has an `ember-cli-deploy` config file?

Reply:
To publish the demo in gh-pages. The configuration is taken from https://github.com/BBVAEngineering/ember-cli-web-components

Reply:
> Don't know if it is intended or a copy/paste thing

Both.